src/views/text-parsing/ui/text-parsing-input.tsx

```diff
@@ -0,0 +1,59 @@
+'use client'
+
+import { Button, Textarea } from '@mantine/core'
+import { useEffect } from 'react'
+import toast from 'react-hot-toast'
+import { useOptionListSuspenseQuery } from '@/entities/attendance-option'
+import { useTextParsingStore } from '@/entities/text-parsing'
+import { AdminContainer, AdminTitle, Icon } from '@/shared/ui'
+import { parseTextSafely } from '../lib'
+
+export const TextParsingInput: React.FC = () => {
+  const { data: attendanceOptions } = useOptionListSuspenseQuery()
+
+  const text = useTextParsingStore((state) => state.text)
+  const options = useTextParsingStore((state) => state.options)
+  const { setText, setOptions, setResult } = useTextParsingStore((state) => state.actions)
+
+  const handleParsingText = () => {
+    if (options) {
+      parseTextSafely({
+        text,
+        parsingOptions: options,
+        attendanceOptions,
+        onSuccess: () => toast.success('분석이 완료됐어요'),
+      })
+    }
+  }
+
+  useEffect(() => {
+    return () => {
+      setText('')
+      setOptions(null)
+      setResult([])
+    }
+  }, [setText, setOptions, setResult])
+
+  return (
+    <AdminContainer>
+      <AdminTitle className="mb-6 flex items-center justify-between">
+        <div className="flex items-center">
+          <Icon query="fluent-emoji:books" className="mr-2" />
+          텍스트 분석
+        </div>
+        <Button onClick={handleParsingText} disabled={text.trim() === ''}>
```

Comment:
[제안]
해당 표현식을 적잖이 사용하신 것으로 파악됩니다. JS에서 빈문자열은 Falsy한 값으로 분류되기 때문에 아래와 같이 수정하는 것이 더욱 깔끔하고 간결하게 코드를 작성할 수 관리할 수 있을 것 같은데 어떠신가요?

<br />


```suggestion
        <Button onClick={handleParsingText} disabled={!text.trim()}>
```

Reply:
넵 좋습니다. 훨씬 깔끔한 것 같아요. 수정하겠습니다.